pipelines/tasks/hisat2.wdl

```diff
@@ -16,7 +16,10 @@ task HISAT2PE {
       --rg PL:ILLUMINA --rg PU:${sample_name} \
       --new-summary --summary-file ${output_name}.log \
       --met-file ${output_name}.hisat2.met.txt --met 5 \
-      -p 4 -S ${output_name}.sam
+      --seed 12345 \
+      -k 10 \
+      --secondary \
+      -p 4 -S ${output_name}.sam 
```

Comment:
If they really do have only short-name parameters, I wouldn't mind having some comments about why each parameter is chosen (in cases where you chose them because "these are suggested to work with RSEM", that's enough of a comment).

Reply:
If they're short name parameters it may be helpful to have some comments explaining what they are.. or use the long names if that's easier?

Reply:
It won't let me comment below, but do we have a policy on how we want the formatting to be with respect to maybe white space between task definitions? I like having one enter between them personally

Reply:
we should be consistent one way or the other, the `picard.wdl` has a `\n` between tasks.  Spacing between task input declaration and different blocks in the task itself are worth making consistent as well.